library/config.c

```diff
@@ -50,12 +50,25 @@ int ziti_load_config(ziti_config *cfg, const char* cfgstr) {
     if (!cfgstr) {
         return ZITI_INVALID_CONFIG;
     }
+    bool seems_like_json = false;
+    const char* c = cfgstr;
+    while (*cfgstr && isspace((unsigned char)*cfgstr)) {
```

Comment:
this should check `*c` not `cfgstr` otherwise it is infinite loop

Reply:
yeah, just so happened that the first char wasn't whitespace when i tested it. nice catch